book.rb

```diff
@@ -0,0 +1,9 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals # this is supposed to be array that houses the rentals made by persons
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+  end
+end
```

Comment:
Kindly add a method for adding rentals inside the book class in order to achieve the has_many association 👍👍😀
Please check this lesson for more information : [here](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship) ✔😀